src/Akka.Hosting.Tests/Akka.Hosting.Tests.csproj

```diff
@@ -4,6 +4,8 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="Microsoft.Extensions.Configuration.Json" Version="7.0.0" />
+    <PackageReference Include="Microsoft.Extensions.Configuration.EnvironmentVariables" Version="7.0.0" />
```

Comment:
Should probably use the `$(MicrosoftExtensionsVersion)` substitution here

Reply:
Done